Gemfile

```diff
@@ -9,6 +9,7 @@ group :development , :test do
   gem 'cancancan'
   gem 'pundit'
   gem 'jruby-openssl', '~> 0.10.1', platform: :jruby
+  gem 'jquery-rails'
```

Comment:
Currently `jquery-rails` is a runtime dependency of activeadmin (it's listed as `add_runtime_dependency` in the gemspec). At the end of this file, there's a `gemspec path: "."` call, which automatically picks up the dependencies from the gemspec. So adding this is not needed, and it will result in the gem being added twice.

Reply:
I have implemented this gems all together. Adding gem here was what worked, The other way around did not wok when I did that.

Reply:
This file is not used at runtime, so I don't see how it could work. Could you show a sample application to prove your point?